Cargo.toml

```diff
@@ -13,7 +13,7 @@ include = ["src/**/*", "Cargo.toml"]
 
 [dependencies]
 bytes = "0.4"
-protobuf = "2.8.1"
+protobuf = "2.10.0"
```

Comment:
can you perhaps fixed it as:
```
protobuf = "= 2.10.0"
```
?

As protobuf seems to require the same version constant, so cargo's automatic semantic minor version bumps won't work?

Reply:
Done